istio-operator/templates/operator-istio-crd.yaml

```diff
@@ -72,17 +89,23 @@ spec:
             galley:
               description: Galley configuration options
               properties:
+                enabled:
+                  type: boolean
                 image:
                   type: string
                 replicaCount:
                   format: int32
                   type: integer
+                resources:
+                  type: object
               type: object
             gateways:
```

Comment:
Since we are bumping to 0.1.12 now, could you please add the new `ports` field to the crd now? 
https://github.com/banzaicloud/istio-operator/blob/release-1.1/config/crds/istio_v1beta1_istio.yaml#L110
https://github.com/banzaicloud/istio-operator/blob/release-1.1/config/crds/istio_v1beta1_istio.yaml#L149

Reply:
Added, thanks @Laci21!